### ISSUE TITLE
After ipfs init, where is the readme located? 

### ISSUE THREAD

**Issue by seandotau** [ipfs/support#51]
So in the docs (https://ipfs.io/docs/getting-started/), I do ipfs init. 
I get a readme and a whole bunch of other files I can look at. 
Check out some of the other files in this directory:
  ./about
  ./help
  ./quick-start     <-- usage examples
  ./readme          <-- this file
  ./security-notes

Where are these actually stored? Can I "view the source" so to speak?

**Comment by ghost** [comment]
Just like you accessed readme -- it's just basic paths:

```
> ipfs cat /ipfs/QmVLDAhCY3X9P2uRudKAryuQFPM5zqA3Yij1dY8FpGbL7T/readme
> ipfs cat /ipfs/QmVLDAhCY3X9P2uRudKAryuQFPM5zqA3Yij1dY8FpGbL7T/quick-start
> ipfs cat /ipfs/QmVLDAhCY3X9P2uRudKAryuQFPM5zqA3Yij1dY8FpGbL7T/about
```

Feel free to close the issue if this answers your question.

**Comment by seandotau** [comment]
Thanks. Let me rephrase. Are the files on my local hard drive? If yes, where are they stored? Is the path for example .ipfs/blocks/2A/readme.txt? (clearly not but as an example).

**Comment by ghost** [comment]
The respective content blocks are stored within `$HOME/.ipfs`, you need to use ipfs as a medium for accessing them though -- larger files get split into 256K block, and the blockstore is optimized for performance so the paths are not available there.

**Comment by seandotau** [comment]
Ok. So it is stored on my local computer in the $HOME/.ipfs directory. Neat. 

Am I able to find for example the text "If you're seeing this, you have successfully installed IPFS and are now interfacing with the ipfs merkledag!" from the read me file then? I would imagine the file is less than 256K so is sitting in a *.data file in the blocks folder somewhere? 

Just starting out with IPFS and trying to get my head around these fascinating new concepts.

**Comment by ghost** [comment]
Theoretically you can grep through all these files, but I really recommend you use the the `ipfs` commands instead

**Comment by seandotau** [comment]
For sure. Just wanted to understand from the ground up. Thanks.
